### PR TITLE
Fixed typos in the documentation

### PR DESCRIPTION
**Pull request by shashi278**

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Our records indicate we have not received your CLA. For legal reasons we need you to sign this before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.

If you have recently signed the CLA, please wait at least one business day
before our records are updated.


You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for your contribution, we look forward to reviewing it!

**Comment by pablogsal** [comment]
Thanks for the PR @shashi278!

For me to merge this you need to fix the CLA. Check the instructions in Bedervere's comment :)

**Comment by shashi278** [comment]
Thanks, @pablogsal ! I signed the CLA as instructed. :)

**Comment by pablogsal** [comment]
Don't worry about the Azure failure, is unrelated

**Comment by pablogsal** [comment]
@shashi278 We need to wait until the CLA signed is registered

File: Parser/pgen/pgen.py
Comment (awecx) [inline Parser/pgen/pgen.py]: Same as https://github.com/python/cpython/pull/15414.
```suggestion
  meaning the rule 'a: b' specifies that a can be replaced by b. A context-free
```

Comment (shashi278) [reply Parser/pgen/pgen.py]: I don't know how to update a PR. That's why I closed #15414 and created a new updated one.

Comment (pablogsal) [reply Parser/pgen/pgen.py]: You need to push more commits to your branch. If you are not familiar with git I can do it for you.

I recommend you to check some tutorials on git and GitHub if you want to know more

Comment (shashi278) [reply Parser/pgen/pgen.py]: Thanks. I figured out that pushing more commits to my branch and creating a pull request afterward would've done the trick. I'm still learning how to work with GitHub. :)

Comment (pablogsal) [reply Parser/pgen/pgen.py]: Notice that this PR is created from your master branch, so you need to put more commits there and push to your remote branch for them to appear here.

Check out https://realpython.com/python-git-github-intro/ for how to do that and what these terms mean

Comment (shashi278) [reply Parser/pgen/pgen.py]: I'm still using it through the website rather than the terminal. So this `remote branch` was indeed confusing to me. I checked that out. Seems like I've got a lot more to explore and learn. Thanks for the direction.

Comment (awecx) [reply Parser/pgen/pgen.py]: You can also hit the "apply code suggestion" from this web page and later pull the changes on your local branch to have this changes reflected.

Comment (shashi278) [reply Parser/pgen/pgen.py]: @aw1621107 Thanks! Committed that.

**Review by epicfaace — APPROVED** [review]
Looks good! Thanks

**Comment by shashi278** [comment]
Thanks! Are we waiting for more reviews?

**Comment by epicfaace** [comment]
Yeah, now a core developer will need to review.

**Review by pablogsal — APPROVED** [review]
LGTM

Thanks @shashi278 for the PR! :)

**Comment by pablogsal** [comment]
Hummmm, there are some CI failures because some unrelated stuff. I will close and reopen the PR to retrigger the CI.

**Comment by pablogsal** [comment]
Congrats @shashi278 for your first commit into CPython! :tada:

**Comment by shashi278** [comment]
Yay! I'm a contributor now.
Thanks to @pablogsal and others for suggestions, support, and encouragement.

**Pull request merged**
